### ISSUE TITLE
Nothing happens or changes once plugin is enabled

### ISSUE THREAD

**Issue by JerkyGunner** [stdscatchemall/HDTCompatabilityWindow#4]
Spoke on Twitter, once plugin is enabled there are no additional windows showing whether Hearthstone is active window or not. Nothing that OBS can detect either.

Running Windows 10 and playing Hearthstone in windowed mode while using updated versions of HDT and the plugin.

**Comment by stdscatchemall** [comment]
To ensure consistency, you have the latest version 0.1.2?

**Comment by JerkyGunner** [comment]
Yes, just updated it this morning.

**Comment by stdscatchemall** [comment]
Can you:

1. Ensure that both DLL's in the Plugins folder are marked as "Unblocked" in their properties?
 - HDTCompatabilityWindow.dll
 - Xceed.Wpf.Toolkit.dll

2. Upload a screenshot of your settings

Windows 10 has nothing to do with it, and it seems to be an isolated event with your system as I cannot replicate the bug.

**Comment by JerkyGunner** [comment]
![image](https://cloud.githubusercontent.com/assets/11801379/9412903/f48e946c-482e-11e5-8ad3-0cbbafe1e19c.png)

Not sure if those are the settings you're referring to, if not just let me know which ones you need specifically.

I'm not sure what you mean by whether they're "unblocked" as there seems to be no option to block/unblock them. As far as I can tell everything is fine and their Properties seem to be fine.

**Comment by Assmodius** [comment]
I'm having the same issue, I think. I've unblocked the .dll files and the plugin is showing up under Tracker/Plugins. I enable it and restart the software. At that point, what do I do to use it? I hit the "Plugins" button on the main window at the far right of the toolbar, but the HDT CW plugin isn't there, only the other plugin I'm using (Arena Helper). Once I've enabled it, what do I do then? How do I use it?

**Comment by stdscatchemall** [comment]
The actual plugin settings are stored within the options, not the Plugins menu like conventional plugins. (In the next release I will add a button that when clicked leads to those settings)
As the window is a copy of the OverlayWindow class from HDT, it will not be shown if the game is not running as it wouldn't make sense to have it running when Hearthstone isn't running. Capturing with OBS will work if you setup the window once hearthstone is open, but you cannot do it before that.

**Comment by JerkyGunner** [comment]
Just to clarify on my end: even with Hearthstone open and a match going, OBS doesn't see any new window or capturable object.

**Comment by stdscatchemall** [comment]
What version of OBS are you using?

**Comment by JerkyGunner** [comment]
Just made some progress: my HDT was set to hide when HS is in the background so it seems whenever I switched to OBS to set up the capture it would become impossible to find. Would it be possible to get this to keep showing on stream even if I set the normal tracker to hide when it's in the background as it doesn't make sense for it to remain open.

Although now I seem to have a black screen over the stream: does that have something to do with the Chroma key?

![image](https://cloud.githubusercontent.com/assets/11801379/9413599/182073f6-4833-11e5-83c8-8fb295f1b363.png)

**Comment by stdscatchemall** [comment]
Yeah. You have to Chroma Key the black away. There was no way to use transparency and a window at the same time. Chroma Key Pure Black, then mess with the Blend Setting. I have mine set to 20 and it looks fine.

As for having that option, I can definitely put that in as an option.

**Comment by JerkyGunner** [comment]
Yeah I just set it up as green and Chroma Key'ed it away.

Only thing I need now is for it to show on stream even when HS is in the background, then it'd be perfect. Thanks for your help on the first issue :)

**Comment by stdscatchemall** [comment]
No problem! :)

@Assmodius Are you still having issues?

**Comment by JerkyGunner** [comment]
Actually scratch that: it seems to still be visible if HS is in the background. It looks like it was only an issue when setting it up initially. I'll keep using it and if I see any more problems I'll let you know.

**Comment by stdscatchemall** [comment]
Oh yeah, that was an issue I fixed while messing around with it once I started to test for possible bugs related to your previous issue. The new version won't do that by default. But it has a setting to override the "HideInBackground" setting

**Comment by JerkyGunner** [comment]
Ok so once the new version is out I'll need to enable that setting.

**Comment by stdscatchemall** [comment]
That is correct. Only reason I put it to not override by default is I don't know how many people want that feature over the alternative. So if more people tell me they want it visible while HS is in the background, I will change it's default to true.

**Comment by Assmodius** [comment]
Yeah, still struggling. Changed the background color to green but I'm not sure what is meant by "Chroma Key'ed it away". I'm broadcasting to twitch and looking at my own stream, but there's no overlay on the stream.

**Comment by stdscatchemall** [comment]
Make sure that you have the Overlay Capture set above your other captures. The Right Click on the window capture, go to properties, and look for a chrome (color) key option.

**Comment by Assmodius** [comment]
Sorry, but you're apparently dealing with a clueless n00b. I'm not sure what you're talking about in your last post. I'm currently running XSplit gamecaster, HDT, and HS itself. I've clearly not done something important, but I'm not sure what.

**Comment by Assmodius** [comment]
I've set the HDT to not hide when HS is in the background, but that did not change things. Like JerkyGunner I'm not seeing any additional windows, so I'm not sure what to do.

**Comment by stdscatchemall** [comment]
Ohhhh, You're XSplit, that would have helped xD I was giving you instructions for OBS.

In your window capture, you've selected the "HDT Compatibility Window" correct?

**Comment by Assmodius** [comment]
I don't believe so. I'm not certain what "In your window capture" means. I've been looking at XSplit and Twitch settings but I'm not seeing anything helpful.

**Comment by Assmodius** [comment]
Part of the problem is that I've never streamed before so I have no frame of reference when trying to talk about it with you.

**Comment by stdscatchemall** [comment]
I recommend that you look up the basic tutorials for using xsplit. There are a lot of good YouTube videos out there that will help you out. I would not be the person you want to talk to about using it.

**Comment by Assmodius** [comment]
The thing is that I'm broadcasting just fine. I can see myself on the stream and HS is displayed fine, it's just that the HDT overlay isn't showing up. That's why I was coming to you for assistance. The broadcast is working.

**Comment by stdscatchemall** [comment]
You need to window capture the window created by my plugin, with the name "HDT Compatibility Window"

**Comment by Assmodius** [comment]
Yeah, that's part of the problem. I'm not getting any window created by your plugin. It's enabled and I'm running HS, but no additional window is appearing.

**Comment by stdscatchemall** [comment]
Update to the latest version and try again. let me know if it doesn't work after that.

**Comment by Assmodius** [comment]
Updated to the latest version and unblocked the new files. Still not seeing an additional window, sadly. Thoughts?

**Comment by Assmodius** [comment]
Ok, I'm getting the overlay window now. What's next?

**Comment by Assmodius** [comment]
Perhaps I should have asked this sooner, but is the free version of XSplit enough to do all of this? I wasn't planning on shelling out cash for this ability.

**Comment by Assmodius** [comment]
Yeah, it's odd. I've got the overlay window now but XSplit isn't broadcasting it. I'm not sure how to resolve that.

**Comment by stdscatchemall** [comment]
If you've got it captured, make sure that the overlay's position is OVER the other windows

**Comment by Assmodius** [comment]
Every time I try to select the overlay window it brings up HS. I can't seem to select the overlay window without it doing that.

**Comment by stdscatchemall** [comment]
The window is designed to stay behind Hearthstone, you don't need it to be visible in order for XSplit to capture it. If you have the window and still don't know how to set it up, I would STRONGLY recommend you go to either the [XSplit Forums](https://support.xsplit.com/forum/) or their [Live Chat Support](https://support.xsplit.com/forum/viewtopic.php?t=5234)

**Comment by Assmodius** [comment]
Yeah, I'm on the XSplit forums now. Thanks for the assistance, sir.
